app/build.gradle

```diff
@@ -44,6 +44,9 @@ android {
             buildConfigField("String", "intercomAppId", "\"\"")
             buildConfigField("String", "amplitudeKey", "\"\"")
         }
+        buildConfigField("boolean", "CRASHLYTICS_ENABLED", "true")
```

Comment:
Sorry but we don't need crashlytics on for debug builds, only for feature and production